.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Set up python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install python build dependencies
+      run: |
+          python -m pip install --upgrade pip build
+
+    - name: Build wheels
+      run: |
+        python -m build --wheel --sdist
+        mkdir wheelhouse
+        mv dist/*.whl wheelhouse/
+    - name: List and check wheels
+      run: |
+        pip install twine pkginfo>=1.11.0
+        ${{ matrix.ls || 'ls -lh' }} wheelhouse/
+        twine check wheelhouse/*
+    - name: Upload wheels
+      uses: actions/upload-artifact@v4
+      with:
+        name: wheels-${{ matrix.python-version }}
+        path: ./wheelhouse/*.whl
```

Comment:
Moving only `*.whl` only copies the wheel to `./wheelhouse`, so the `.tar.gz` isn't getting uploaded, I think. Can we just upload whatever is in `dist/`?